runtime/lua/vim/lsp/protocol.lua

```diff
@@ -738,11 +738,7 @@ function protocol.make_client_capabilities()
       completion = {
         dynamicRegistration = false,
         completionItem = {
-          -- Until we can actually expand snippet, move cursor and allow for true snippet experience,
-          -- this should be disabled out of the box.
-          -- However, users can turn this back on if they have a snippet plugin.
-          snippetSupport = false,
-
+          snippetSupport = true,
```

Comment:
Not sure if this is the only capability change or if there's another completion capability I need to set.

Reply:
That's the one I was thinking of, but if there are still parts missing, we don't need to do it yet. I'll just keep mentioning this on PRs until we can :)

Reply:
Yeah I think we should do this in a later PR since I think it will require a larger change in `vim.lsp.util` (that might involve some discussion).